docs/how_to/single_node.md

```diff
@@ -1 +1,152 @@
-# How To Guides
+# Deploying a single node
+
+Deploying a single-node cluster is a great way to experiment with M3DB and get a feel for what it
+has to offer. Our Docker image by default configures a single M3DB instance as one binary
+containing:
+
+- An M3DB storage instance (`m3dbnode`) for timeseries storage. This includes an embedded tag-based
+  metric index, as well as as an embedded etc server for storing the above mentioned cluster
+  topology and runtime configuration.
+- A "coordinator" instance (`m3coordinator`) for writing and querying tagged metrics, as well as
+  managing cluster topology and runtime configuration.
+
+To begin, first start up a Docker container with port `7201` (used to manage the cluster topology)
+and port `9003` (used to read and write metrics) exposed. We recommend you create a persistent data
+directory on your host for durability:
+
+```
+docker run -p 7201:7201 -p 9003:9003 --name m3db -v $(pwd)/m3db_data:/var/lib/m3db quay.io/m3db/m3db:latest
+```
+
+<!-- TODO: link to docs containing explanations of what namespaces, the coordinator,
+placements, etc. are -->
+
+<!-- TODO: add something about how this is in no way a recommended production deployment guide,
+and write a guide for what is considered a production-ready deployment (this is in the works) -->
+
+Next, create an initial namespace for your metrics:
+
+```json
+curl -X POST localhost:7201/namespace/add -d '{
+  "name": "default",
+  "options": {
+    "bootstrapEnabled": true,
+    "flushEnabled": true,
+    "writesToCommitLog": true,
+    "cleanupEnabled": true,
+    "snapshotEnabled": false,
+    "repairEnabled": false,
+    "retentionOptions": {
+      "retentionPeriodNanos": 172800000000000,
+      "blockSizeNanos": 7200000000000,
+      "bufferFutureNanos": 600000000000,
+      "bufferPastNanos": 600000000000,
+      "blockDataExpiry": true,
+      "blockDataExpiryAfterNotAccessPeriodNanos": 300000000000
+    },
+    "indexOptions": {
+      "enabled": true,
+      "blockSizeNanos": 7200000000000
+    }
+  }
+}'
+```
+
+With a namespace to hold your metrics created, you can initialize your first placement:
+
+```json
+curl -X POST localhost:7201/placement/init -d '{
+    "num_shards": 64,
+    "replication_factor": 1,
+    "instances": [
+        {
+            "id": "m3db_local",
+            "isolation_group": "rack-a",
+            "zone": "embedded",
+            "weight": 1024,
+            "endpoint": "127.0.0.1:9000",
+            "hostname": "127.0.0.1",
+            "port": 9000
+        }
+    ]
+}'
+```
+
+Shortly after, you should see your node complete bootstrapping! Don't worry if you see warnings or
+errors related to a local cache file. Those are expected for a local instance.
```

Comment:
Maybe we should note which errors/issues are expected? Would be annoying if the user received a legit error, but thought it was benign bc of this comment.